ogusa/aggregates.py

```diff
@@ -369,3 +369,43 @@ def resource_constraint(Y, C, G, I, K_f, new_borrowing_f,
                 debt_service_f)
 
     return rc_error
+
+
+def get_K_splits(B, K_demand_open, D_d, zeta_K):
+    r'''
+    Returns total domestic capital as well as amounts domestica capital
```

Comment:
@jdebacker . This is really a nit pick, but I found it/saw it, so here it is. Get rid of extra "a" in "domestica" misspelling.

Reply:
Ok - just pushed fix to typo in this branch. Thanks for catching that @rickecon.

Reply:
@jdebacker I will merge this as soon as it finishes passing all the checks, which it will in about 20 minutes. Thanks.